webpack/assets/javascripts/react_app/routes/Models/constants.js

```diff
@@ -6,7 +6,7 @@ export const MODELS_PAGE_HIDE_LOADING = 'MODELS_PAGE_HIDE_LOADING';
 export const MODELS_PAGE_SHOW_LOADING = 'MODELS_PAGE_SHOW_LOADING';
 export const MODELS_PAGE_CLEAR_ERROR = 'MODELS_PAGE_CLEAR_ERROR';
 
-export const MODELS_SEARCH_PROPS = getControllerSearchProps('models');
+export const MODELS_SEARCH_PROPS = getControllerSearchProps('models','models');
```

Comment:
is this change needed, or is it a leftover from when testing it on the models page?

Reply:
Not needed, nice catch!